tsconfig.base.json

```diff
@@ -61,6 +61,7 @@
       "@fxa/shared/pem-jwk": ["libs/shared/pem-jwk/src/index.ts"],
       "@fxa/shared/react": ["libs/shared/react/src/index.ts"],
       "@fxa/shared/sentry": ["libs/shared/sentry/src/index.ts"],
+      "@fxa/shared/sentry/client": ["libs/shared/sentry/src/client.ts"],
```

Comment:
Some modules in `shared-sentry` require modules only available in a `node` environment, such as `fs`, which are not available on the client (browser). Therefore a separate `client` barrel file is required which would not include these modules.